src/viewer/Viewer.jsx

```diff
@@ -23,19 +23,27 @@ const isCordova = () => window.cordova !== undefined
 
 export default class Viewer extends Component {
   componentDidMount() {
+    this.fetchMore()
```

Comment:
I am not sure the `fetchMore` is under the responsibility of the viewer?

The other solution is to keep the whole code in the `FilesViewer` component, in its `render()` function.

What do you think?